packages/test-utils/src/harness.ts

```diff
@@ -34,11 +31,12 @@ export async function createTestHarness(
   if (sessionId !== undefined) {
     // Session-based transport
     transportOptions.generateSessionId = () => sessionId;
-    transportOptions.eventStore = eventStore || new InMemoryEventStore();
-  } else if (eventStore !== undefined) {
+    transportOptions.sessionStore =
```

Comment:
wondering: is there any way to disable sessions entirely?

Reply:
you can disable by passing `undefined` to the `generateSessionId` callbacl